### ISSUE TITLE
Tflite convertor inserted additional 1x1 conv layers

### ISSUE THREAD

**Issue by yadonskov** [tensorflow/tensorflow#42188]
@tensorflow/micro

**System information**
- Windows 10
- pip install tensorflow==2.1.0

**Describe the problem**
I made a model with tf.keras, converted it using TensorflowLite and got strange redundant layers 4 1x1 conv layers.

**Please provide the exact sequence of commands/steps when you ran into the problem**
This is my script

`
import tensorflow as tf
from tflite_runtime.interpreter import Interpreter
import numpy as np
import os
unixToolsPaths=r'C:\ARC_2019_12\tools\make;C:\ARC_2019_12\tools\cmake\bin;C:\ARC_2019_12\tools\usr\x86_64-pc-msys\bin;C:\ARC_2019_12\tools\usr\bin;'
os.environ['PATH'] += unixToolsPaths
import subprocess

def makeModel():
    inputs = tf.keras.Input(shape=(49, 10, 1))
    x = tf.keras.layers.Conv2D( 64, (10,4), (2, 2), padding='same', activation='relu')(inputs)
    x = tf.keras.layers.SeparableConv2D( 64, (3,3), padding='same')(x)
    x = tf.keras.layers.Conv2D( 64, (1,1), padding='valid', activation='relu')(x)
    x = tf.keras.layers.SeparableConv2D( 64, (3,3), padding='same')(x)
    x = tf.keras.layers.Conv2D( 64, (1,1), padding='valid', activation='relu')(x)
    x = tf.keras.layers.SeparableConv2D( 64, (3,3), padding='same')(x)
    x = tf.keras.layers.Conv2D( 64, (1,1), padding='valid', activation='relu')(x)
    x = tf.keras.layers.SeparableConv2D( 64, (3,3), padding='same')(x)
    x = tf.keras.layers.Conv2D( 64, (1,1), padding='valid', activation='relu')(x)
    x = tf.keras.layers.AveragePooling2D((25, 5))(x)
    x = tf.keras.layers.Flatten()(x)
    x = tf.keras.layers.Dense(2)(x)
    x = tf.keras.layers.Softmax()(x)
    model = tf.keras.Model(inputs=inputs, outputs=x)
    return model

def quantizationDataGenerator():
    for i in range(10):
        yield [ np.random.rand(1, 49, 10, 1).astype(np.float32)]

def convertTFL(name, model):
    converter = tf.lite.TFLiteConverter.from_keras_model(model)
    converter.optimizations = [tf.lite.Optimize.DEFAULT]
    converter.representative_dataset = quantizationDataGenerator
    converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
    converter.inference_input_type = tf.int8
    converter.inference_output_type = tf.int8
    quantModel = converter.convert()

    with open("{}.tflite".format(name), "wb") as f:
        f.write(quantModel)

def convertToCC(file, ccFile):
    subprocess.run(r'xxd -i {} > {}'.format(file, ccFile), shell=True, check=True)
    
    with open(ccFile, 'r') as f:
        linesOrig = f.readlines()
    lines = [
        'unsigned char modelBuffer[] DATA_ALIGN_ATTRIBUTE = {\n'
    ]
    lines.extend(linesOrig[1:-1])
                
    with open(ccFile, 'w') as f:
        f.writelines(lines)
        

if __name__ == '__main__':
    name = 'kws'
    model = makeModel()
    model.summary() 
    convertTFL(name, model)
`

**Comment by Saduf2019** [comment]
@yadonskov 
I ran the code shared it is not indented, please provide with complete indented code such that we can replicate the issue faced or if possible share a colab gist with the error faced.

**Comment by yadonskov** [comment]
@Saduf2019 
https://gist.github.com/yadonskov/5eba06542c9e83329d9a10b84ea5233f#file-tf_issue-ipynb

**Comment by yadonskov** [comment]
I reproduced same behaviour in 2.3 version too

**Comment by Saduf2019** [comment]
@yadonskov 
I ran the code shared please find the [gist here](https://colab.research.google.com/gist/Saduf2019/28f84982b95bb046d9b510c667af2e9b/untitled367.ipynb), can you please point out the extra layer you are referring to the model summary and code both lead to 14 layers as expected.

**Comment by yadonskov** [comment]
When I see in Netron - I see additional like double 1x1 convolutions 
![image](https://user-images.githubusercontent.com/17152311/91968848-efb9ff00-ed1d-11ea-9544-7a5485fed1df.png)
maybe Netron visualization is incorrect?
